engine/src/main/java/org/terasology/config/flexible/FlexibleConfig.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2016 MovingBlocks
```

Comment:
Just noticed you are still outputting 2016 copyright notices. I think a running `gradlew idea` might do the trick.